infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -423,11 +367,9 @@ public synchronized void resetServer(CorfuPayloadMsg<SequencerTailsRecoveryMsg>
         // It is necessary because we reset the sequencer.
         if (!bootstrapWithoutTailsUpdate) {
             // Evict all entries from the cache. This eviction triggers the callback modifying the maxConflictWildcard.
-            conflictToGlobalTailCache.cleanUp();
-
+            cache.invalidateAll();
```

Comment:
Why is this changed from cleanUp to invalidateAll required?

Reply:
because `cleanUp` doesn't delete all records it just triggers `maintenance` logic

Reply:
You can also install a new cache.

Reply:
Recreate cache is a very expensive operation, I just measured it's about 5 times slower than invalidateCache

Reply:
Yep. The cache does a few reflective calls on create (e.g. class load the optimal entry factory for minimal memory overhead). Some of that could be faster but unfortunately there was a JDK memory leak with Method Handles. Regardless things are optimized favoring the assumption that the cache will probably be long-lived.